fixtures/beamz/ps10w-led-pin-spot-10w-4-in-1-dmx.json

```diff
@@ -0,0 +1,123 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "PS10W LED PIN SPOT 10W 4-IN-1 DMX",
+  "shortName": "PS10W",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["Raine M. Ekman"],
+    "createDate": "2024-05-30",
+    "lastModifyDate": "2024-05-30"
+  },
+  "links": {
+    "manual": [
+      "https://www.beamzlighting.com/wp-content/uploads/2024/05/151.259-PS10W-LED-Pin-Spot-10W-RGBW-DMX_manual_V1.3.pdf"
+    ],
+    "productPage": [
+      "https://www.beamzlighting.com/product/ps10w-led-pin-spot-10w-4-in-1-dmx/"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=Mb6_FTCzp1o"
+    ]
+  },
+  "physical": {
+    "dimensions": [68, 145, 150],
+    "weight": 0.7,
+    "power": 17,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED"
+    }
+  },
+  "availableChannels": {
+    "Red": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "White": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    },
+    "Dimmer": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 15],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [16, 255],
+          "type": "Intensity"
+        }
+      ]
+    },
+    "Strobe": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 15],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [16, 255],
+          "type": "StrobeSpeed",
```

Comment:
```suggestion
          "type": "ShutterStrobe",
          "shutterEffect": "Strobe",
```

Reply:
I disagree, as `StrobeSpeed` is documented as "global, doesn't activate strobe directly", and my understanding is that this channel both activates the strobe and sets the speed.

Reply:
Right, this channel at 16-255 enables the strobe and sets the speed, and at 0-15 disables the strobe (which we could have also written as `"type": "ShutterStrobe", "shutterEffect": "Open"`).

Without this channel as a `ShutterStrobe` capability, how would you enable/disable the strobe?